src/test/java/uk/gov/pay/adminusers/utils/DatabaseTestHelper.java

```diff
@@ -371,9 +371,9 @@ public DatabaseTestHelper insertServiceEntity(ServiceEntity serviceEntity) {
             return handle.createStatement("INSERT INTO services(" +
                     "id, name, custom_branding, " +
                     "merchant_name, merchant_telephone_number, merchant_address_line1, merchant_address_line2, merchant_address_city, " +
-                    "merchant_address_postcode, merchant_address_country, merchant_email, external_id) " +
+                    "merchant_address_postcode, merchant_address_country, merchant_email, external_id, redirect_to_service_immediately_on_terminal_state) " +
                     "VALUES (:id, :name, :customBranding, :merchantName, :merchantTelephoneNumber, :merchantAddressLine1, :merchantAddressLine2, " +
-                    ":merchantAddressCity, :merchantAddressPostcode, :merchantAddressCountry, :merchantEmail, :externalId)")
+                    ":merchantAddressCity, :merchantAddressPostcode, :merchantAddressCountry, :merchantEmail, :externalId, :redirect_to_service_immediately_on_terminal_state)")
```

Comment:
Probably `:redirect_to_service_immediately_on_terminal_stat` should be camel case